src/components/cards/collection-card.tsx

```diff
@@ -54,8 +56,12 @@ export class CollectionCard extends React.Component<IProps> {
             size='40px'
             unlockWidth
           />
-          <TextContent>{this.getCertification(repo)}</TextContent>
-          <SignatureBadge isCompact signState={sign_state} />
+
+          <div className='left-container'>
```

Comment:
```suggestion
          <div className='hub-left-container'>
```

Reply:
though given the css, this should really be
```suggestion
          <Flex>
```